core/ts.h

```diff
@@ -227,6 +227,8 @@ class TransitionSystem
   // getters
   const smt::SmtSolver & solver() const { return solver_; };
 
+  smt::SmtSolver & get_solver() { return solver_; };
```

Comment:
Ahh yeah this was fairly ugly. So if I change the return type of `solver` I get const related compile error messages. What do you think? I'm not sure the right way to work around this.

Reply:
Oh yes, I remember we had a similar discussion on `const`some time ago. Changing this will require many more updates in other classes. So let's keep it.